charts/victoria-metrics-alert/README.md

```diff
@@ -115,6 +115,11 @@ Change the values according to the need of the environment in ``victoria-metrics
 | alertmanager.configMap | string | `""` |  |
 | alertmanager.enabled | bool | `false` |  |
 | alertmanager.extraArgs | object | `{}` |  |
+| alertmanager.extraArgs | object | `{}` |  |
```

Comment:
It seems like `extraArgs` is now duplicated, could you please remove the duplicate?

Reply:
fix in fda37843f64e8b2179470067910ba4fdfadd34c5